packages/codemirror/style/index.css

```diff
@@ -25,7 +25,6 @@
   font-size: var(--jp-code-font-size);
   font-family: var(--jp-code-font-family);
   height: auto;
-  background: transparent;
```

Comment:
What is the reason for the back and forth on this property? I removed it in #2788 since it broke the vim command line, but couldn't really figure out what it was for. @cameronoelsen indicated to me that it was unnecessary.